tfconfig/module.go

```diff
@@ -22,12 +23,29 @@ type Module struct {
 	Diagnostics Diagnostics `json:"diagnostics,omitempty"`
 }
 
-func newModule(path string) *Module {
+type ProviderAliases map[string][]string
```

Comment:
It might be helpful to have a comment explaining this type, especially since the key and value are interchangeable.